src/server/announcements.ts

```diff
@@ -58,17 +58,29 @@ export class Announcements {
   }
 
   async announce (actor: string): Promise<void> {
+    const published = new Date().toUTCString()
+    const to = ['https://www.w3.org/ns/activitystreams#Public']
+    const cc = [`${this.actorUrl}followers`]
     const activity = {
       '@context': 'https://www.w3.org/ns/activitystreams',
-      type: 'Note',
+      type: 'Create',
       id: `${this.outboxUrl}/${nanoid()}`,
       actor: this.actorUrl,
-      attributedTo: this.actorUrl,
-      published: new Date().toUTCString(),
-      to: ['https://www.w3.org/ns/activitystreams#Public'],
-      cc: [`${this.actorUrl}followers`],
-      // TODO: add a template in config
-      content: `a wild site appears! ${actor}`
+      published,
+      to,
+      cc,
+      object: {
+        '@context': 'https://www.w3.org/ns/activitystreams',
+        type: 'Note',
+        id: `${this.outboxUrl}/${nanoid()}/note`,
```

Comment:
I think we can change this to be `#note` then the json-ld magic will resolve it? I'll test like this first.

Reply:
I know nothing about json-ld :shrug: but sounds good to me